mmdeploy/codebase/mmcls/models/classifiers/base.py

```diff
@@ -35,9 +35,11 @@ def base_classifier__forward(
     if self.head is not None:
         output = self.head(output)
 
-    from mmcls.models.heads import MultiLabelClsHead
+    from mmcls.models.heads import ConformerHead, MultiLabelClsHead
```

Comment:
Could we add test for `conformer` in https://github.com/open-mmlab/mmdeploy/blob/dev-1.x/tests/regression/mmcls.yml?